static/css/base/headings.less

```diff
@@ -106,6 +106,13 @@ h4 {
     font-family: @lucida_sans_serif-1;
     font-weight: normal;
   }
+  &.work-subtitle {
```

Comment:
The base/headings.less file is meant to define common heading styles. This particular style is however not generic- it's part of the lists component.

Please move to static/css/components/mybooks-list.less

Reply:
Done.